odiglet/cmd/main.go

```diff
@@ -41,9 +41,17 @@ func main() {
 		panic(err)
 	}
 
-	// If started in init mode
-	if len(os.Args) == 2 && os.Args[1] == "init" {
-		odiglet.OdigletInitPhase(clientset)
+	// Check if started in a special mode
+	// The mode can be passed as any argument, not necessarily the first one
+	for _, arg := range os.Args {
+		switch arg {
+		case "init":
+			odiglet.OdigletInitPhase(clientset)
+			return
```

Comment:
previous to this change, the "init" mode would not return, but now it does. Is it intentional?